src/fastMoneyRound.js

```diff
@@ -4,35 +4,46 @@ import Game from './game';
 class FastMoneyRound extends Round {
   constructor(survey, player1, player2) {
     super(survey, player1, player2)
+    this.timerId = 0
   }
 
   startTime(game) {
```

Comment:
This is getting called in a `setTimeout`, which could be causing an issue with when the clearTimeout(timerId) is getting called.